components/automate-deployment/pkg/history/logrus_interceptor.go

```diff
@@ -0,0 +1,35 @@
+package history
+
+import (
+	"github.com/sirupsen/logrus"
+)
+
+type logrusInterceptor struct{}
+
+func NewLogrusInterceptor() EventInterceptor {
+	return logrusInterceptor{}
+}
+
+func (logrusInterceptor) LogStart(ev *StartEvent) {
+	logrus.WithFields(logrus.Fields{
+		"id":          ev.ID,
+		"class":       ev.Name,
+		"description": ev.Description,
+		"cause":       ev.Cause,
+		"tags":        ev.Tags,
+	}).Info("Event Start")
```

Comment:
[nit] In your example output, this ends up putting the information I care about pretty far into the log line.  I wonder if we want to make `ev.Description` the thing we pass to `Info`?  I see the value in driving this towards something more machine readable than our current logs but I worry it is gonna make debugging harder in the short term.

Reply:
we can.